test/Libraries/RevitIntegrationTests/SelectionTests.cs

```diff
@@ -612,6 +613,115 @@ public void SelectElementsByCategoryInView()
         }
 
 
+        [Test]
+        [TestModel(@".\Selection\DynamoSample.rvt")]
+        public void SelectModelElementsByCategory()
+        {
+            OpenAndAssertNoDummyNodes(Path.Combine(workingDirectory, @".\Selection\SelectModelElementsByCategory.dyn"));
+            TestMultipleCategorySelection<Element, Element>();
+        }
+ 
+        [Test]
+        [TestModel(@".\SampleModel.rvt")]
+        public void SelectRoomsByStatus()
+        {
+            OpenAndAssertNoDummyNodes(Path.Combine(workingDirectory, @".\Selection\SelectRoomsByStatus.dyn"));
+
+            RunCurrentModel();
+
+            var selectNode =
+                ViewModel.Model.CurrentWorkspace.FirstNodeFromWorkspace<RevitNodeModel>();
+            Assert.NotNull(selectNode);
+
+            var expectedElementsCategoryString = "Rooms";
+            var outputDictionary = GetPreviewValue(selectNode.GUID.ToString());
+            Assert.NotNull(outputDictionary);
+
+            var placedRooms = GetPreviewCollection("e512c62255eb4a349ce7126af54ef03d");
+            var unplacedRooms = GetPreviewCollection("ae185814931b4acfbe92d38a9abfc79b");
+            var notEnclosedRooms = GetPreviewCollection("b3959f94df634a3787ae421dc8e5f409");
+            var redundantRooms = GetPreviewCollection("bc195cf9592b4e7bb8574414ae149f1b");
+
+            Assert.NotNull(placedRooms);
+            Assert.NotNull(unplacedRooms);
+            Assert.NotNull(notEnclosedRooms);
+            Assert.NotNull(redundantRooms);
+
+            AssertElementCategoryOnCollection(placedRooms, expectedElementsCategoryString);
+            AssertElementCategoryOnCollection(unplacedRooms, expectedElementsCategoryString);
+            AssertElementCategoryOnCollection(notEnclosedRooms, expectedElementsCategoryString);
+            AssertElementCategoryOnCollection(redundantRooms, expectedElementsCategoryString);
+
+        }
+
+
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void SelectModelElementById()
```

Comment:
@AndyDu1985 This test use element ID as a hard code input for node "Element by Id", I don't think it will stable when RevitAPI changes. Do you have any suggestions about this node's test?